x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -508,9 +517,18 @@ public boolean isActive() {
     public boolean checkFeature(Feature feature) {
         boolean allowed = isAllowed(feature);
         LongAccumulator maxEpochAccumulator = lastUsed.get(feature);
+        long now = System.currentTimeMillis();
         if (maxEpochAccumulator != null) {
             maxEpochAccumulator.accumulate(epochMillisProvider.getAsLong());
         }
+
+        if(feature.minimumOperationMode.compareTo(OperationMode.BASIC) > 0
+            && now > status.licenseExpiryDate - GRACE_PERIOD_DURATION.getMillis()) {
+            HeaderWarning.addWarning("Your license will expire in [{}] days. " +
+                    "Contact your administrator or update your license for continued use of features",
```

Comment:
I'm not sure about the wording here but I'll defer to a native speaker and maybe get Fabio's input too? 

Maybe s/update/renew ?  and s/for continued use of/in order to continue using the licensed ?

Reply:
This is the wording Kibana has. I talked to Fabio and he suggested us to be consistant with wording when possible.

Reply:
SGTM then!